run-build-functions.sh

```diff
@@ -449,6 +450,13 @@ install_dependencies() {
     fi
   fi
 
+  # Homebrew from Brewfile
+  if [ -f Brewfile.netlify ] || [ ! -z "$HOMEBREW_BUNDLE_FILE" ]
+  then
+    echo "Installing Homebrew dependencies from ${HOMEBREW_BUNDLE_FILE:-Brewfile.netlify}"
+    brew bundle --file Brewfile.netlify
```

Comment:
i'm not sure what happens if you set `HOMEBREW_BUNDLE_FILE` to something custom
what do you think about making `Brewfile.netlify` a default value for `HOMEBREW_BUNDLE_FILE` inside the if?

Reply:
oh yeah.